cyences_app_for_splunk/default/savedsearches.conf

```diff
@@ -2733,7 +2733,40 @@ action.cyences_notable_event_action.attacker_drilldown = `cs_o365` sourcetype="o
 action.cyences_send_email_action = 1
 action.cyences_notable_event_action.products = Office 365
 
-
+[O365 - OneDrive or SharePoint File Sharing with External User]
+disabled = 1
+enableSched = 1
+alert.track = 1
+alert.severity = 4
+alert.suppress = 0
+counttype = number of events
+quantity = 0
+relation = greater than
+cron_schedule = 4 * * * *
+description = This alert will pinpoint when file sharing with external user usein Onedrive and Sharepoint. \
+\
+Data Collection - Office 365 management activity data (Splunk Add-on for Office 365). 
+dispatch.earliest_time = -4h@h
+dispatch.latest_time = +4h@h
+display.general.type = statistics
+display.page.search.tab = statistics
+display.page.search.mode = fast
+request.ui_dispatch_app = cyences_app_for_splunk
+request.ui_dispatch_view = search
+search = `cs_o365` index=o365 sourcetype="o365:management:activity" _index_earliest=-61m@m _index_latest=-1m@m Workload IN (OneDrive, SharePoint) Operation IN ("SharingSet", "AccessRequestApproved", "SharingRestored", "AnonymousLinkCreated", "SharingInvitationCreated", "SharingInvitationUpdated", "AddedToSecureLink", "SharingPolicyChanged") NOT TargetUserOrGroupType IN ("Member", "SharePointGroup") \
+| stats values(Operation) AS Operations, values(EventData) AS EventData, values(ItemType) as ItemType, values(ApplicationDisplayName) as ApplicationDisplayName, values(EventSource) as EventSource latest(_time) as LastTime earliest(_time) as FirstTime by SourceFileName, UserId \
+|rename UserId as User \
+| eval cyences_severity = if(is_privileged_user=="Yes", "critical","high") \
```

Comment:
this line should be after | `cs_user_privilege_mapping(User)`  as is_privileged_user field is populating from the macro.